install_mj_source.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+while getopts h: flag; do
+    case "${flag}" in
+        h) hash=${OPTARG};;  # Hash of the MuJoCo commit to install
+    esac
+done
+
+echo -e "\n[NOTE] Installing mujoco from source..."
+
+# the script directory and mujoco directory
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+mujoco_dir="$script_dir/../mujoco"
```

Comment:
Highly optional: it might be nice to allow the user to specify a different place to install mujoco

Reply:
Good point. I'll expose this in the next commit.